engine-sdk/src/near_runtime.rs

```diff
@@ -388,7 +392,7 @@ pub(crate) mod exports {
         fn account_balance(balance_ptr: u64);
         pub(crate) fn attached_deposit(balance_ptr: u64);
         pub(crate) fn prepaid_gas() -> u64;
-        fn used_gas() -> u64;
+        pub(crate) fn used_gas() -> u64;
```

Comment:
This change doe not look necessary

Reply:
It was after tests. Will remove.

Reply:
Shouldn't clippy be catching this? If not, I know that it can be configured. I'll add an issue about making a clippy file.